### ISSUE TITLE
Bug: Video on course page is not screen reader accessible

### ISSUE THREAD

**Issue by briangrossman** [mitodl/mitxonline#512]
### Steps to Reproduce

- Try to access the video with a screen reader

### Expected Behavior

- The user should be able use the video player with the screen reader and be able to identify all the buttons.
- Note the the video player on the [xPRO product pages](https://xpro.mit.edu/programs/program-v1:xPRO+SysEngx/) works fine. 

### Actual Behavior

- Nope. Details below from our testers

---

### Details from testing

In the process of adding captions to this player, screen reader access was broken. 

Take this one https://mitxonline.mit.edu/courses/course-v1:MITxT+14.100x/ 

The ARC Toolkit shows the player has an invalid ARIA-LABEL of "YouTube Video Player".
![image](https://github.mit.edu/storage/user/923/files/dca4f180-b98c-11ec-8562-a84573abafcc)

I noticed this, sent it to @rjc for testing, and Rich shared this: "I have no idea if this is a YouTube embedded player, or something created by mitX or another third party. The mitX video on this page somehow disables all screen reader navigation within the player region after the play button is pressed. It also throws focus out of the region when you first click the play button, so in order to control the player you must:

- find the region in the document while the audio is playing (not always easy since audio covers up screen reader speech)
- tab into the player region since all screen reader navigation within the region is turned off (in fact, all content within the region seems to be hidden)
- tab till you find the pause button and press spacebar
- Once this is done, you can tab around inside the player region and you can use YouTube shortcuts like "k" to play/pause and arrows to move around.

There are buttons which are not labeled correctly; I think they are "full screen" and "mute". The are announced as "button" when they gain focus, and when they lose focus, they are announced correctly.  Basically, they have aria-live on these buttons for no reason I can understand; removing that and making sure that text within the button is visible, or using aria-label="text of label" on the button will suffice.

Again, I cannot figure out why the contents of the player region cannot be normally navigated. Is HTML being injected into an embedded YouTube player?   

For reference, here is a typical YouTube embed for a random mitX video:

```
<iframe width="560" height="315" src="https://www.youtube.com/embed/YLf3eX-VkzQ" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen>
</iframe>
```

If you put this into a document, all controls show up via screen reader and navigation is fairly straightforward. YouTube has it's querks, but they are well understood by screen reader users at this point."

---

### Related Issues

https://github.com/mitodl/mitxonline/pull/491

**Comment by umarmughal824** [comment]
The video player is being rendered by a third-party package (https://github.com/videojs/videojs-youtube )that is ARIA-LABEL of "YouTube Video Player". In my opinion that should not be an issue as everything seems to be working fine.
